functions-methods.js

```diff
@@ -16,6 +40,40 @@
 // typeOfEmail("novi.nlaapjesk@outlook.com") geeft geeft "Extern" <-- deze moet het ook doen!
 // typeOfEmail("a.wiersma@outlook.com") geeft "Extern"
 
+function assignment2() {
+  const emails = [
+    "n.eeken@novi-education.nl",
+    "t.mellink@novi.nl",
+    "novi.nlaapjesk@outlook.com",
+    "a.wiersma@outlook.com",
+  ];
+
+  for (i = 0; i < emails.length; i++) {
+    let domain = typeOfEmail(emails[i]);
+
+    console.log(domain);
+  }
+}
+
+function typeOfEmail(emailAdress) {
+  let domain = emailAdress.substring(
+    emailAdress.indexOf("@") + 1,
+    emailAdress.lastIndexOf(".")
+  );
+
+  let result = "";
+
+  if (domain === "novi") {
+    result = "Student";
+  } else if (domain === "novi-education") {
+    result = "Medewerker";
+  } else {
+    result = "Extern";
+  }
+
+  return result;
```

Comment:
Als je na het if-else-block niets meer met de waarde gaat doen, dan kun je de waarde ook meteen teruggeven:
```suggestion
  if (domain === "novi") {
    return "Student";
  } else if (domain === "novi-education") {
    return "Medewerker";
  }
  
  return "Extern";
```